regression-test/suites/nereids_function_p0/scalar_function/U.groovy

```diff
@@ -43,4 +43,5 @@ suite("nereids_scalar_fn_U") {
 	qt_sql_upper_String_notnull "select upper(kstr) from fn_test_not_nullable order by kstr"
 	sql "select user() from fn_test"
 	sql "select user() from fn_test_not_nullable"
+    qt_sql_url_decode "select url_decode('https%3A%2F%2Fdoris.apache.org%2Fzh-CN%2Fdocs%2Fsql-manual%2Fsql-functions%2Fstring-functions')"
```

Comment:
The indentation needs to be aligned like other SQL

Reply:
> The indentation needs to be aligned like other SQL


![image](https://github.com/apache/doris/assets/48236177/34841cf7-ccaf-4c67-b5c6-ce6b6758a094)
This may be a display error. Source files are indented normally

Reply:
no display error，maybe you indented  \t or ' '

Reply:
> no display error，maybe you indented \t or ' '

Emm... My fault.